Towny/src/main/java/com/palmergames/bukkit/towny/command/ResidentCommand.java

```diff
@@ -176,6 +177,9 @@ public List<String> onTabComplete(CommandSender sender, Command command, String
 					if (args.length == 3)
 						return Collections.singletonList("[page #]");
 					break;
+				case "outlawlist":

```

Comment:
this line can get bumped downwards, just above or below line with `case "tax":` on it since it uses the same output.